components/MapsDedi/index.vue

```diff
@@ -1,215 +1,171 @@
-                            'maps__boxmaps-sidebar-content-list-village-item-info-detail-list-level--blue': village.level === 3,
-                            'maps__boxmaps-sidebar-content-list-village-item-info-detail-list-level--yellow': village.level === 2,
-                            'maps__boxmaps-sidebar-content-list-village-item-info-detail-list-level--pink': village.level === 1
-                          }"
-                        >
-                          Level {{ village.level ? village.level : '' }}
-                        </div>
-                        <div v-if="village.category.name" class="maps__boxmaps-sidebar-content-list-village-item-info-detail-list-tema">
-                          {{ village.category.name }}
-                        </div>
-                      </div>
+                  <div class="maps__boxmaps-sidebar-content-list-village-item-info-detail-city">
+                    {{ village.city.name }}
+                  </div>
+                  <div class="maps__boxmaps-sidebar-content-list-village-item-info-detail-list">
+                    <div
+                      :class="{
+                        'maps__boxmaps-sidebar-content-list-village-item-info-detail-list-level': true,
+                        'maps__boxmaps-sidebar-content-list-village-item-info-detail-list-level--green': village.level === null || village.level === 4,
+                        'maps__boxmaps-sidebar-content-list-village-item-info-detail-list-level--blue': village.level === 3,
+                        'maps__boxmaps-sidebar-content-list-village-item-info-detail-list-level--yellow': village.level === 2,
+                        'maps__boxmaps-sidebar-content-list-village-item-info-detail-list-level--pink': village.level === 1
+                      }"
+                    >
+                      Level {{ village.level ? village.level : '' }}
                     </div>
-                    <div class="maps__boxmaps-sidebar-content-list-village-item-image">
-                      <img class="rounded-lg object-cover h-full w-full" :src="[village.images.length ? village.images[0] : require('~/assets/logo/logo-dedi-abu-abu.svg')]" alt="gambar sebaran desa digital">
+                    <div v-if="village.category.name" class="maps__boxmaps-sidebar-content-list-village-item-info-detail-list-tema">
+                      {{ village.category.name }}
                     </div>
                   </div>
-                  <div class="maps__boxmaps-sidebar-content-list-village-item-separator" />
                 </div>
-                <div ref="observer" class="h-10 w-full" />
+                <div class="maps__boxmaps-sidebar-content-list-village-item-image">
+                  <img class="rounded-lg object-cover h-full w-full" :src="[village.images.length ? village.images[0] : require('~/assets/logo/logo-dedi-abu-abu.svg')]" alt="gambar sebaran desa digital">
```

Comment:
hello @adzharamrullah , sorry for asking this question,
I just wondering what is the purpose of using `require('...')` on the image `src`? 🤔

Reply:
hello @Ibwedagama the reason because i use `:src` / **attribute binding**  / **directive** not 'src' **native attribute** it's mean without `required('...')` webpack alias cannot identify path with attribute binding, so best practice for resolve that problem is using `required('...')`

Reply:
Oh I didn't know about that, thank you for the insight @adzharamrullah 👍🏼

Reply:
Okay, your welcome ❤️